### ISSUE TITLE
ReferenceError: path is not defined

### ISSUE THREAD

**Issue by peteruithoven** [ryanmurakami/webpack-addons-react#4]
When running this with webpack-cli `1.3.3` the generated webpack.config.js has issues, like it uses path, but doesn't require it. 

```
$ webpack
/home/peteruithoven/Projects/www/webpack/webpack3/webpack-cli-init-react/webpack.config.js:8
  context: path.join(__dirname, "src"),
           ^

ReferenceError: path is not defined
    at Object.<anonymous> (/home/peteruithoven/Projects/www/webpack/webpack3/webpack-cli-init-react/webpack.config.js:8:12)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at requireConfig (/usr/lib/node_modules/webpack/bin/convert-argv.js:97:18)
    at /usr/lib/node_modules/webpack/bin/convert-argv.js:104:17
```

**Comment by a-x-** [comment]
ping

**Comment by ryanmurakami** [comment]
Thanks for opening the issue @peteruithoven! I pushed a fix and it should be resolved in `0.1.5`. Let me know if you have any other issues.
